content/en/docs/operator/introduction.md

```diff
@@ -10,7 +10,7 @@ This guide describes the Kubeflow Operator and the current supported releases of
 
 Kubeflow Operator helps deploy, monitor and manage the lifecycle of Kubeflow. Built using the [Operator Framework](https://coreos.com/blog/introducing-operator-framework) which offers an open source toolkit to build, test, package operators and manage the lifecycle of operators.
 
-The operator is currently in incubation phase and is based on this [design doc](https://docs.google.com/document/d/1vNBZOM-gDMpwTbhx0EDU6lDpyUjc7vhT3bdOWWCRjdk/edit#). It is built on top of _KfDef_ CR, and uses _kfctl_ as the nucleus for Controller. Current roadmap for this Operator is listed [here](https://github.com/kubeflow/kfctl/issues/193). The Operator is also [published on OperatorHub](https://operatorhub.io/operator/kubeflow).
+The operator is currently in incubation phase and is based on this [design doc](https://docs.google.com/document/d/1vNBZOM-gDMpwTbhx0EDU6lDpyUjc7vhT3bdOWWCRjdk/edit#). It is built on top of _KfDef_ CR (Custom Resource), and uses _kfctl_ as the nucleus for Controller. Current roadmap for this Operator is listed [here](https://github.com/kubeflow/kfctl/issues/193). The Operator is also [published on OperatorHub](https://operatorhub.io/operator/kubeflow).
```

Comment:
Thanks for suggesting this change! The convention in tech docs when introducing an abbreviation is to use the full form of the term with the abbreviation in parenthesis. Would you please update this line as follows? 

CR (Custom Resource) -> custom resources (CR)

Also, line 65 also introduces this abbreviation. Would you please make the following change to line 65:

custom resources (CR) -> CR